samples/Hack/main

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env hhvm
+<?hh // strict
```

Comment:
We prefer sample files from "real" applications since they are used to train the Bayesian classifier. Could you also mention in the original post the origin and license of the sample once you've changed it?